sdk/spring/scripts/replace_util.py

```diff
@@ -21,56 +21,56 @@
     'appconfiguration': {
         'sdk/appconfiguration/azure-spring-cloud-test-appconfiguration-config/pom.xml': {
             VERSION_UPDATE_ITEMS: (
-                VersionUpdateItem('org.springframework.boot:spring-boot-starter-parent', '2.2.9.RELEASE'),
+                VersionUpdateItem('org.springframework.boot:spring-boot-starter-parent', '2.2.11.RELEASE'),
```

Comment:
@jialindai , @saragluna , @yiliuTo , @ZhijunZhao , FYI:

Even though we claim that our starter can support `2.2.x` and `2.3.x` at the same time.
But in this case, `2.2.11` work well, but `2.2.9` will have error.

It's caused by netty/reactor-netty does not backward compatible even for it's minor version up.